README.md

```
@@ -9,32 +9,80 @@ For more information about the Inspector project please read [Avo documentation]
+import { AvoInspector, AvoInspectorEnv } from "react-native-avo-inspector";
 
-let inspector = new Inspector.AvoInspector({
+let inspector = new AvoInspector({
+  apiKey: "your api key",
+  env: AvoInspectorEnv.Dev,
+  version: "1.0.0",
+  appName: "My app",
+});
+```
+
+### With encryption
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)